manticore/seth.py

```diff
@@ -1189,7 +1189,7 @@ def finalize(self):
         for address, md in self.metadata.items():
             with self._output.save_stream('global_%s.sol'%md.name) as global_src :
                 global_src.write(md.source_code)
-            with self._output.save_stream('global_%s.runtime_bytecode'%md.name) as global_runtime_bytecode :
+            with self._output.save_stream('global_%s_runtime.bytecode'%md.name) as global_runtime_bytecode :
```

Comment:
sorry in #654 i was unclear, but we should also do this for the .init_bytecode file

Reply:
we don't want to keep this as `.bytecode` as well for symmetry?

Reply:
yes, both runtime and init bytecode should end in .bytecode